plugins/processors/awsappsignals/internal/resolver/kubernetes.go

```diff
@@ -685,6 +685,11 @@ func (h *kubernetesHostedInAttributeResolver) Process(attributes, resourceAttrib
 		attributes.PutStr(attr.HostedInClusterNameK8s, h.clusterName)
 	}
 
+	//The application log group in Container Insights is a fixed pattern:
+	// "/aws/containerinsights/{Cluster_Name}/application"
```

Comment:
Since we don't have this as a shared constant in code anywhere, could you add a link to where it's defined in the operator?  I suppose that would be https://github.com/aws/amazon-cloudwatch-agent-operator/blob/fe144bb02d7b1930715aa3ea32e57a5ff13406aa/helm/templates/fluent-bit-configmap.yaml#L82 .

Reply:
Sure,  I will add it.